crawler.js

```diff
@@ -463,9 +497,17 @@ class Crawler {
     return true;
   }
 
+  createPage(url){
+    var id = uuidv4();
+    var today = new Date();
+    var row = {"id": id, "url": url.href, "title": page.title()}
+    this.pages.push(row)
+  }
+  
   shouldCrawl(url) {
     try {
       url = new URL(url);
+      this.createPage(url)
```

Comment:
Probably shouldn't call createPage() here as this function just checks is a URL should be crawled..
The best place to add it might be in the task function or in the driver, but then more work in the driver code..